HISTORY.rst

```diff
@@ -3,8 +3,21 @@
 History
 -------
 
+v0.23 (2019-04-04)
```

Comment:
you've somehow messed up this file I'm afraid, I guess by rebasing wrongly.

Reply:
Should be ok now